site/content/docs/03-run-time.md

```diff
@@ -475,13 +475,48 @@ TODO
 * fade, fly, slide, scale, draw
 * crossfade...
 
-### `svelte/animation`
+### `svelte/animate`
 
-TODO
+The `svelte/animate` module exports one function for use with svelte [animations](docs#Animations).
+
+#### `flip`
+
+```sv
+animate:flip={params}
+```
+
+The `flip` function animates elements from their start position to their end position across `x` and `y` axes.
+
+`flip` accepts the following parameters:
+
+* `delay` (`number`, default 0) — milliseconds before starting
+* `duration` (`number` | `function`, default `d => Math.sqrt(d) * 120`) — see below
```

Comment:
` — see below` could be ` — milliseconds, see below`

Reply:
A function isn't milliseconds, this would be incorrect in my eyes. `milliseconds or function returning milliseconds` could work but wouldn't match other descriptions. It would need to be `- duration of the animation in milliseconds or function returning the duration of the animation in milliseconds` but this is long and confusing which is why I pointed to the longer explanation below.